tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react-jsx"
```

Comment:
Do we need this change? I'm unable to compile the project with this

Reply:
it was not added by me. it looks its added when i updated the typescript